power/nut-client/patches/replace_system.patch

```diff
@@ -14,19 +14,17 @@
  
 +static int runcmd(const char *cmd)
```

Comment:
i guess the new patch is from upstream?

Reply:
No, this isn't upstreamed. I just noticed the exit code does not match how `system` behaves. I can move this into a separate PR if needed.